content-script.js

```diff
@@ -1,16 +1,32 @@
 (async function enhance() {
-    const { items } = await chrome.storage.sync.get(['items'])
+    const { items } = await chrome.storage.sync.get(['items']);
+    let replacements = [
+        //merge request title
+        document.querySelector('h1')
+    ];
+
+    //merge request description
+    let title = document.getElementsByClassName('description');
+
+    if(title.length) {
+        replacements.push(title[0]);
+    }
```

Comment:
is it possible to replace it by this ?
 ```suggestion
    const replacements = [
        // merge request title
        document.querySelector('h1'),
        // merge request description
        document.querySelector('description'),
    ].filter(Boolean);
```